### PR TITLE
fix: Markdownlint

### LINKED ISSUE #692: [Connector KIT] Fix Markdown Linting Issues
Due to the activation of the markdown linter in #674, several problems with different markdown files exist that need to be fixed.

Output:
```
docs-kits/kits/Connector Kit/Adoption View/adoption-view.md:8 MD025/single-title/single-h1 Multiple top-level headings in the same document [Context: "# Adoption View"]
docs-kits/kits/Connector Kit/Adoption View/adoption-view.md:86:119 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs-kits/kits/Connector Kit/Adoption View/adoption-view.md:153:1 MD053/link-image-reference-definitions Link and image reference definitions should be needed [Unused link or image reference definition: "dsp-url"] [Context: "[dsp-url]: https://docs.intern..."]
docs-kits/kits/Connector Kit/Development View/architecture.md:24 MD025/single-title/single-h1 Multiple top-level headings in the same document [Context: "# Architecture"]
docs-kits/kits/Connector Kit/Development View/architecture.md:46:91 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs-kits/kits/Connector Kit/Development View/architecture.md:53:118 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs-kits/kits/Connector Kit/Development View/specifications.md:6 MD025/single-title/single-h1 Multiple top-level headings in the same document [Context: "# Specifications"]
docs-kits/kits/Connector Kit/Development View/specifications.md:23:115 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs-kits/kits/Connector Kit/Development View/specifications.md:24:172 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs-kits/kits/Connector Kit/Operation View/operation-view.md:6 MD025/single-title/single-h1 Multiple top-level headings in the same document [Context: "# Operation View"]
docs-kits/kits/Connector Kit/Operation View/operation-view.md:33:109 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
docs-kits/kits/Connector Kit/Operation View/operation-view.md:40:65 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

### PR DESCRIPTION
**Pull request by bcronin90**
Fix markdownlint issues

Closes #692 

MD053 in adoption-view.md looks like a false positive to me.

### CODE REVIEW

File: docs-kits/kits/Connector Kit/Adoption View/adoption-view.md
Comment (stephanbcbauer) [inline docs-kits/kits/Connector Kit/Adoption View/adoption-view.md:3]: @bcronin90 have you checked if the title is used for the header 1? Or if it's the ID? Did you build it locally?

image

File: docs-kits/kits/Connector Kit/Development View/architecture.md
Comment (stephanbcbauer) [inline docs-kits/kits/Connector Kit/Development View/architecture.md:2]: same here, i am not sure if the title is used for the headline?

**Review by stephanbcbauer — DISMISSED** [review]
LGTM, but please check if the title or id is used for header generation. THX

**Review by mhellmeier — CHANGES_REQUESTED** [review]
The build of the website fails which should be fixed before merging.

File: docs-kits/kits/Connector Kit/Operation View/operation-view.md
Comment (stephanbcbauer) [inline docs-kits/kits/Connector Kit/Operation View/operation-view.md:2]: is -> id

**Comment by bcronin90** [comment]
The "title" field is used for the headline, which is why it caused duplication warnings in the linter. The "id" field is purely for under-the-hood stuff. While I was removing the titles, I added ids where there weren't any. Apparently that broke something somewhere. So I'll revert that.

**Review by SebastianBezold — APPROVED** [review]

**Comment by bcronin90** [comment]
So what about this one?
```
docs-kits/kits/Connector Kit/Adoption View/adoption-view.md:152:1 MD053/link-image-reference-definitions Link and image reference definitions should be needed [Unused link or image reference definition: "dsp-url"] [Context: "[dsp-url]: https://docs.intern.../"]
```
Turns out it's not a false positive, just a bit misleading. We had a wrong bracket type in there. Yay for linting.

**Comment by bcronin90** [comment]
Okay, all Markdown errors should be from outside our scope now.

**Comment by bcronin90** [comment]
Might be something in one of the sidebars js'

File: docs-kits/kits/Connector Kit/Development View/architecture.md
Comment (mhellmeier) [inline docs-kits/kits/Connector Kit/Development View/architecture.md:2]: Removing the title will cause a wrong display in the menu.

**Review by mhellmeier — CHANGES_REQUESTED** [review]
This PR removes the title in different files. Therefore, it doesn't get displayed with an uppercase starting letter. This has nothing to do with the sidebar file.

**Comment by bcronin90** [comment]
> This PR removes the title in different files. Therefore, it doesn't get displayed with an uppercase starting letter. This has nothing to do with the sidebar file.

So what's the best practice here? Cause if I add it again, I get the duplicate header warning from the markdown linter, which this was supposed to remove. I can remove the actual markdown header instead. But instead of stumbling in the dark some more, I'll just ask: What's our general approach here?

**Comment by mhellmeier** [comment]
I suggest having a look at all the other KITs. Most KITs (what I would suggest) are using `title` along with `id` and `description`. Example from the Traceability KIT:
```
id: Architecture View Traceability Kit
title: Architecture View
description: 'Traceability Kit'
sidebar_position: 2
```

Please keep in mind to use a unique ID (not just `Architecture`), otherwise the build won't work.

**Comment by stephanbcbauer** [comment]
Not sure, is there still something todo here?

**Comment by bcronin90** [comment]
Ah. I wasn't aware we explicitly linked each document again. I'll fix those links.

**Review by mhellmeier — APPROVED** [review]
Thanks for the fix!

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Pull request merged**
